data/src/main/java/com/teamagam/gimelgimel/data/map/adapter/GeoEntityDataMapper.java

```diff
@@ -37,24 +37,25 @@ public GeoEntity transform(String id, GeoContentData geoContentData) {
     }
 
     public ImageEntity transformIntoImageEntity(String id, PointGeometryData point) {
-        ImageEntity imageEntity = new ImageEntity(id, null, mGeometryMapper.transform(point));
+        ImageEntity imageEntity = new ImageEntity(id, null, mGeometryMapper.transform(point),
+                false);
```

Comment:
Now I finished the CR and I think maybe we can avoid it but I need your guidance. Let's talk about it :)